library/common-library/templates/_custom-attributes.tpl

```diff
@@ -0,0 +1,29 @@
+{{/*
+This function allows easily to overwrite custom attributes to the function "common.customAttributes"
+*/}}
+{{- define "common.customAttributes.overrideAttributes" -}}
+{{- end }}
+
+
+
+{{/*
+This will render custom attributes as a YAML ready to  be templated or be used with `fromYaml`.
```

Comment:
```suggestion
This will render custom attributes as a YAML ready to  be templated or be used with `fromYaml`.
Chart writers can override `common.customAttributes.overrideAttributes`, which will be included in the output of this helper.
```